faq.md

```diff
@@ -46,6 +46,17 @@ Fundamentally, Flutter is for users that want beautiful apps,
 with delightful motion and animation, and UIs with character
 and an identity all their own.
 
+### How experienced of a programmer/developer do I have to be to use Flutter?
+
+Flutter is approachable to programmers familiar with object-oriented
+concepts (classes, methods, variables, etc) and imperative programming
+concepts (loops, conditionals, etc).
+
+No prior mobile experience is required in order to learn and use Flutter.
+
+We have seen people with very little programming experience learn
+and use Flutter for prototyping and app development.
```

Comment:
maybe link to the I/O talk about designers using flutter?

Reply:
yes, that would be good!

Reply:
done